server/src/models/DogPark.js

```diff
@@ -22,6 +22,20 @@ class DogPark extends Model {
             }
         })
     }
+
+    static get relationMappings () {
```

Comment:
nit picking - no space before the parameter parenthesis
https://github.com/airbnb/javascript#whitespace--around-keywords